### ISSUE TITLE
Broken link to Python3 include files building Python3 OpenSCAP bindings on selected distributions

### ISSUE THREAD

**Issue by iankko** [OpenSCAP/openscap#328]
See the thread for further details:
&nbsp; &nbsp; [1] https://www.redhat.com/archives/open-scap-list/2016-February/msg00002.html

**Comment by isimluk** [comment]
This really sucks. But we will perhaps need to copy-paste boilerplate check for python2.

Even though, this was not intended by autoconf* authors, AM_CHECK_PYTHON_HEADERS now checks for python3 on selected distributions.

**Comment by redhatrises** [comment]
@jan-cerny has this been fixed consider we have moved to python3?

**Comment by jan-cerny** [comment]
@redhatrises Nice catch!

On `maint-1.3` branch and `master` branch we are "Python3-only", and we use CMake there, so it is solved there in a different way.

On `maint-1.2` branch I believe this has been solved by https://github.com/OpenSCAP/openscap/pull/990 and https://github.com/OpenSCAP/openscap/pull/997

I have checked that building Python bindings on `maint-1.2` branch works on RHEL8, which is Python3-only distribution. Let's consider it fixed.
